looker_deployer/commands/deploy_content.py

```diff
@@ -125,8 +133,56 @@ def import_content(content_type, content_json, space_id, env, ini, debug=False):
         win_exec = ["cmd.exe", "/c"]
         gzr_command = win_exec + gzr_command
 
+    is_new_dash = "false"
+    existing_dash_alerts = []
+    existing_elements_w_alerts = []
+
+    if content_type == "dashboard":  ## only run for dashboards, looks can't have alerts
+        sdk = get_client(ini, env)  ## should we update the function def and pass sdk?
+        with open(content_json) as file:
+            json_dash = json.load(file)
+        existing_dash = sdk.search_dashboards(slug=json_dash['slug'])  ## search with slug first, fall back to name + folder
+        if len(existing_dash) < 1:
+            existing_dash = sdk.search_dashboards(title=json_dash['title'],folder_id=space_id)
+            if len(existing_dash) < 1:
+                is_new_dash = "true"
+        if is_new_dash == "false": ## if it's an existing dashboard, save the alerts and elements
+            for element in existing_dash[0]['dashboard_elements']:
+                start = len(existing_dash_alerts)
+                alerts = list(filter(lambda alert: alert['dashboard_element_id'] == element['id'], enabled_alerts))
+                if len(alerts) > 0:
+                    existing_dash_alerts.extend(alerts)
+                if len(existing_dash_alerts) > start:
+                    if element not in existing_elements_w_alerts:
+                        existing_elements_w_alerts.append(element)
+
+
+    #logger.debug("space info", extra={"space_name": space_name, "parent_id": parent_id})
     subprocess.run(gzr_command)
 
+    
+    if is_new_dash == "false" and content_type == "dashboard" and len(existing_dash_alerts) > 0:  ## get the new dashboard
+        updated_dash = sdk.search_dashboards(slug=json_dash['slug'])
+        if len(updated_dash) < 1:
+            updated_dash = sdk.search_dashboards(title=json_dash['title'],folder_id=space_id)
+        old_to_new_ids = {}
+        for element in existing_elements_w_alerts:  ## match old element ids to new element ids
+            updated_dash_element = list(filter(lambda item: item['title'] == element['title'] and item['query_id'] == element['query_id'], updated_dash[0]['dashboard_elements']))
+            if len(updated_dash_element) == 1:  ## what should we do if more than one match?
+                old_to_new_ids[element['id']] = updated_dash_element[0]['id']
+        for alert in existing_dash_alerts:  ## create new alerts
+            logger.debug('processing alert for element', extra={"element_id": alert['dashboard_element_id']})
+            new_alert = {}
+            for key in alert.keys():
+                new_alert[key] = alert[key]                
+            if alert['dashboard_element_id'] in old_to_new_ids.keys():
+                logger.debug("creating alert", extra={"old_element_id": alert['dashboard_element_id'], "new_element_id": old_to_new_ids[alert['dashboard_element_id']]})
```

Comment:
Can we get an logger.info line that summarizes what is happening that specifies the dashboard element ID and alert being updated.